python-package/xgboost/compat.py

```diff
@@ -72,6 +73,18 @@ def lazy_isinstance(instance: Any, module: str, name: str) -> bool:
     XGBStratifiedKFold = None
 
 
+def is_cudf_installed():
+    """Check cuDF installed or not"""
+    # Checking by `importing` instead of check `importlib.util.find_spec("cudf") is not None`
```

Comment:
I'm not sure if it's a good idea to hide an important error.

Reply:
what about print a warning here ?

Reply:
Sounds good.